openedx/core/djangoapps/schedules/models.py

```diff
@@ -15,6 +15,9 @@ class Schedule(TimeStampedModel):
     """
 
     enrollment = models.OneToOneField('student.CourseEnrollment', null=False, on_delete=models.CASCADE)
+    # The active field on the schedule is deprecated, please do not rely on it.
+    # You can use the is_active field on the CourseEnrollment model instead (i.e. schedule.enrollment.is_active).
+    # Removing this field from the database is a TODO for https://openedx.atlassian.net/browse/AA-574.
```

Comment:
are there additional details that I should be including in this ticket?

Reply:
Another option for discouraging use would be to rename the field here to like `_active` or `_active_deprecated` and adding a `db_column='active'` kwarg. That way no one would use it accidentally.

That would also let you add an `@property` for `active` that simply returns `self.enrollment.is_active`, but might as well fix all uses rather than leaving a shim I guess. Doesn't seem to be many uses.